src/components/fields/password-field/password-field.js

```diff
@@ -90,7 +105,7 @@ class PasswordField extends React.Component {
             onBlur={this.props.onBlur}
             onFocus={this.props.onFocus}
             isAutofocussed={this.props.isAutofocussed}
-            isPasswordVisible={this.props.isPasswordVisible}
+            isPasswordVisible={this.state.isPasswordVisible}
```

Comment:
Is the initialState missing?

Reply:
I guess it would "coerce" to false from undefined. A `state = { isPasswordVisible: false }` would make that explicit.

Reply:
Yes, it's another thing that I forgot!